tests/xlsx.test.js

```diff
@@ -39,6 +39,11 @@ describe('does xlsx have the right headers', () => {
         const missingHeaders = getMissingColumnHeaders('report_vellore.xlsx');
         expect(missingHeaders).toStrictEqual([]);
     });
+
+    test('ap', () => {
+        const missingHeaders = getMissingColumnHeaders('report_vellore.xlsx');
+        expect(missingHeaders).toStrictEqual([]);
+    });
```

Comment:
looks redundant, unless we change `report_vellore` to `report_ap`. Same for chennai.
<img width="509" alt="image" src="https://github.com/vatz88/FFCSonTheGo/assets/16202323/32d1290c-fae4-4c25-a94b-cdc53783a262">

Reply:
I initially had the same thought; however, considering that I am a novice and this is my first open source contribution, I wondered if I might be mistaken in my assessment.

Reply:
Yeah, this was my mistake Thanks for fixing!